content/influxdb/v2.1/reference/internals/data-retention.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Data retention in InfluxDB
+description: >
+  The InfluxDB retention service checks for and removes data with timestamps beyond
+  the defined retention period of the bucket the data is stored in.
+weight: 103
+menu:
+  influxdb_2_1_ref:
+    name: Data retention
+    parent: InfluxDB internals
+influxdb/v2.1/tags: [storage, internals]
+related: 
+  - /influxdb/v2.1/reference/internals/shards/
+  - /influxdb/v2.1/reference/internals/storage-engine/
+---
+
+The **InfluxDB retention enforcement service** checks for and removes data with
+timestamps beyond the defined retention period of the
+[bucket](/influxdb/v2.1/reference/glossary/#bucket) the data is stored in.
+This service is designed to automatically delete "expired" data and optimize disk
+usage without any user intervention.
+
+By default, the retention enforcement service runs every 30 minutes, but you can
+custom-configure how often the service runs with the
+[`storage-retention-check-interval`](/influxdb/v2.1/reference/config-options/#storage-retention-check-interval)
+configuration option.
+
+- [Bucket retention period](#bucket-retention-period)
+- [Shard group duration](#shard-group-duration)
+- [When does data actually get deleted?](#when-does-data-actually-get-deleted)
+
+## Bucket retention period
+A **bucket retention period** is the duration of time that a bucket retains data.
+Retention periods can be as short as an hour or infinite.
+[Points](/influxdb/v2.1/reference/glossary/#point) in a bucket with timestamps
+beyond the defined retention period (relative to now) are _eligible_ for deletion.
+
+## Shard group duration
+InfluxDB stores data on disk in [shards](/influxdb/v2.1/reference/glossary/#shard).
+Each shard belongs to a shard group and each shard group has a shard group duration.
+The **shard group duration** defines the duration of time or interval that each
```

Comment:
> duration of time or interval that each

What does "or interval" mean here? Is it different from a duration? Is it explanatory, i.e. ", or interval,"? Is it redundant?

Reply:
```suggestion
The **shard group duration** defines the duration of time that each
```